doc/develop/test/pytest.rst

```diff
@@ -67,7 +67,7 @@ Give access to a DeviceAdapter type object, that represents Device Under Test.
 This fixture is the core of pytest harness plugin. It is required to launch
 DUT (initialize logging, flash device, connect serial etc).
 This fixture yields a device prepared according to the requested type
-(native posix, qemu, hardware, etc.). All types of devices share the same API.
+(native, qemu, hardware, etc.). All types of devices share the same API.
```

Comment:
would this be even better?

```suggestion
(``native``, `` qemu``, ``hardware``, etc.). All types of devices share the same API.
```

Reply:
Done. 
But note I'm queuing a *lot* of similarly trivial changes, if we end up polishing around those changes, it will take a very long time.

Reply:
that's why I said the comment was non blocking :)
thanks!